Azure_Pipelines/azure-pipelines.terminology.cd.yaml

```
@@ -0,0 +1,100 @@
+resources:
+  pipelines:
+  - pipeline: pipeline-trigger
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)